pkgs/applications/office/kmymoney/default.nix

```diff
@@ -22,5 +31,6 @@ stdenv.mkDerivation rec {
     homepage = http://kmymoney2.sourceforge.net/;
     description = "KDE personal money manager";
     inherit (kdelibs.meta) platforms maintainers;
+    priority = 100;
```

Comment:
Please add a comment explaining the need for this. Packages typically don't set their own priority, and in fact, `meta.priority` is broken.